Lib/test/test_msvcrt.py

```diff
@@ -0,0 +1,85 @@
+import os
+import sys
+import unittest
+
+from test.support import os_helper
+from test.support.os_helper import TESTFN, TESTFN_ASCII
+
+if sys.platform != "win32":
+    raise unittest.SkipTest("windows related tests")
+
+import _winapi
+import msvcrt
+
+
+class TestFileOperations(unittest.TestCase):
+    def test_locking(self):
+        with open(TESTFN, "w") as f:
+            self.addCleanup(os_helper.unlink, TESTFN)
+
+            msvcrt.locking(f.fileno(), msvcrt.LK_LOCK, 1)
+            self.assertRaises(OSError, msvcrt.locking, f.fileno(), msvcrt.LK_NBLCK, 1)
+
+    def test_unlockfile(self):
+        with open(TESTFN, "w") as f:
+            self.addCleanup(os_helper.unlink, TESTFN)
+
+            msvcrt.locking(f.fileno(), msvcrt.LK_LOCK, 1)
+            msvcrt.locking(f.fileno(), msvcrt.LK_UNLCK, 1)
+            msvcrt.locking(f.fileno(), msvcrt.LK_LOCK, 1)
+
+    def test_setmode(self):
+        with open(TESTFN, "w") as f:
+            self.addCleanup(os_helper.unlink, TESTFN)
+
+            msvcrt.setmode(f.fileno(), os.O_BINARY)
+            msvcrt.setmode(f.fileno(), os.O_TEXT)
+
+    def test_open_osfhandle(self):
+        h = _winapi.CreateFile(TESTFN_ASCII, _winapi.GENERIC_WRITE, 0, 0, 1, 128, 0)
+        self.addCleanup(os_helper.unlink, TESTFN_ASCII)
+
+        fd = msvcrt.open_osfhandle(h, os.O_RDONLY)
+        os.close(fd)
+
+    def test_get_osfhandle(self):
+        with open(TESTFN, "w") as f:
+            self.addCleanup(os_helper.unlink, TESTFN)
+
+            msvcrt.get_osfhandle(f.fileno())
+
+
+class TestConsoleIO(unittest.TestCase):
+    def test_kbhit(self):
+        msvcrt.kbhit()
```

Comment:
```suggestion
        self.assertEqual(msvcrt.kbhit(), 0)
```
The doc says that this returns True on success, implying that it returns False on failure, and that the proper test would be `self.assertIs(msvcrt.kbhit, False), which would fail.  Is the doc wrong in claiming that a bool is returned or is the code buggy in not converting the int to bool before returning?

Reply:
From the clinic declares, the result should be a int: https://github.com/python/cpython/blob/3e53ac99038920550358c1ea0212c3907a8cb385/PC/msvcrtmodule.c#L221 , so I think the document is wrong.